ts/Dashboards/Component/Component.ts

```diff
@@ -113,19 +113,31 @@ abstract class Component<TEventObject extends Component.EventTypes = Component.E
     public options: Component.ComponentOptions;
     public type: string;
     public id: string;
+    // An array of options marked as editable by the UI.
```

Comment:
why dont we use the JSDOC structure for these comments?

```js
/**
 * Option description
 *
 * name
 */

Reply:
This issue is more about removing the old one- we are going to tackle this later